.github/workflows/ship.yaml

```diff
@@ -170,20 +171,57 @@ jobs:
             declare "CARGO_TARGET_${TARGET_SCREAMING}_LINKER"='${{ matrix.linker }}'
             export "CARGO_TARGET_${TARGET_SCREAMING}_LINKER"
           fi
+          # If we're on a tag, use the tag as the release version.
```

Comment:
Does this actually work? I would have thought you'd have to do something like `git tag --contains HEAD` or somesuch.